go.mod

```diff
@@ -200,8 +203,11 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
+// Temporary replace rule until sqldb is tagged.
+replace github.com/lightningnetwork/lnd/sqldb => ./sqldb
```

Comment:
I think we could also solve this with a `go.work` file now, see https://github.com/lightningnetwork/lnd/pull/8562/commits/ed18daa15bdacff873982fb63dbb5ec9fe745fd5 as an example. But non-blocking, as this will be removed in the follow-up PR anyway.

Reply:
Yeah I thought it's just temporary so perhaps no need to introduce go.work here, but thanks for mentioning nonetheless! :)